flixel/system/debug/Console.hx

```diff
@@ -112,7 +112,7 @@ class Console extends Window
 		_input.addEventListener(FocusEvent.FOCUS_OUT, onFocusLost);
 		_input.addEventListener(KeyboardEvent.KEY_DOWN, onKeyPress);
 		
-		#if (cpp || neko) // workaround for broken TextField focus on native
```

Comment:
Yes, by "just work" I meant that I suspected that perhaps it was the old (cpp/neko legacy) hacks that were causing the weird behavior in next, but it appears simplying guarding them off with ifdefs doesn't result in correct behavior.